### ISSUE TITLE
winjs.js , base.js , ui.js ?

### ISSUE THREAD

**Issue by iangithub** [winjs/winjs#1244]
what's difference winjs.js and base.js & ui.js ? 

I've created an Windows 10 UWP project with JavaScript and use Winjs , but when I test some winjs ui controls ,that not run well .

But , if I try remove winjs.js , and include base.js & ui.js , and run again , it run very well .

**Comment by rigdern** [comment]
What version of WinJS are you using?

You should never use all 3 in a single project (winjs.js, base.js, and ui.js).

In WinJS 4.0.1 (the latest version), you should use base.js and ui.js like this (also described in the `Usage` section of http://try.buildwinjs.com/#get):

```js
<script src="winjs/js/base.js"></script>
<script src="winjs/js/ui.js"></script>
```

In some versions of WinJS (3 and 4.0.0-preview), base.js and ui.js were combined into a single file, WinJS.js. However, that is no longer the case in WinJS 4.0.0 and newer.

**Comment by iangithub** [comment]
Hi rigdern ; 
Thank you .
When i create default UWP project , that use version 4.0 and include winjs.js file , but the ui control  not run well , so i try upgrade to version 4.0.1 and still include winjs.js file , but still not run well , so i try remove winjs.js from html page , and try include base.js & ui.js ( like win 8.1 store app ) and run it well , so i am so confuse

**Comment by rigdern** [comment]
TLDR: In WinJS 4.0.1, you should include `base.js` and `ui.js` in your project.

Here's some history in case you are interested in getting a better understanding of `base.js`/`ui.js` vs `WinJS.js`.

When WinJS was created, it contained 2 JavaScript source files both of which must be included to get the full functionality of WinJS: `base.js` and `ui.js`.

In WinJS 3.x, we started experimenting with combining `base.js` and `ui.js` into a single `WinJS.js` file. WinJS 3.x came with all 3 files. You could either include `base.js` and `ui.js` or you could include `WinJS.js`.

WinJS 4.0.0-preview shipped with only `WinJS.js`.

In the the official stable release of WinJS 4 (4.0.0, 4.0.1), we decided to switch back to 2 files: `base.js` and `ui.js`.

Here's a summary of which versions of WinJS ship with which files:

               | base.js | ui.js | WinJS.js
------------ | ------------- | ------------ | -------------
WinJS 1.x | x | x |
WinJS 2.x | x | x |
WinJS 3.x | x | x | x
WinJS 4.0.0-preview | | | x
WinJS 4.x | x | x |

**Comment by iangithub** [comment]
Hi rigdern ; 

Thank you :) .
